jnius_config.py

```diff
@@ -66,11 +66,27 @@ def get_classpath():
     "Retrieves the classpath the JVM will use."
     from os import environ
     from os.path import realpath
+    import sys
     global classpath
 
     # add a path to java classes packaged with jnius
-    from pkg_resources import resource_filename
-    return_classpath = [realpath(resource_filename(__name__, 'jnius/src'))]
+    if sys.version_info >= (3, 9):
+        from contextlib import ExitStack
+        import importlib.resources, atexit
+
+        # see https://importlib-resources.readthedocs.io/en/latest/migration.html#pkg-resources-resource-filename
+        file_manager = ExitStack()
+        atexit.register(file_manager.close)
+        # importlib.resources.files is only available from Python 3.9
+        # use https://github.com/python/importlib_resources/issues/60 not __name__ as jnius_config.py is not a package
+        resource_path = importlib.resources.files('jnius') / 'src'
+        return_classpath = [ str(resource_path) ]
+        path = file_manager.enter_context(
+            importlib.resources.as_file(resource_path))
```

Comment:
And, if we do not care about `path`, do we need `file_manager` ?

Reply:
`path` was apparently just return of the context manager. I dont think we need it.
However, `file_manager` is needed as the resource may be part of a zip file, and if so, it will have been extracted and cleaned up at exit. I removed `path` from updated code.

Further information in the linked importlib-resources URL.